### PR TITLE
Remove cancel option for DEPUBLISH

### PR DESCRIPTION
**Pull request by andyjmaclean**

### CODE REVIEW

File: src/app/dashboard/ongoingexecutions/ongoingexecutions.component.ts [hunk 1]
```diff
@@ -45,6 +46,18 @@ export class OngoingexecutionsComponent implements OnInit {
...
+  /*  @param {WorkflowExecution} ongoing - the execution
+  */
+  canCancelWorkflow(ongoing: WorkflowExecution): boolean {
+    return (
+      [`${PluginType.PUBLISH}`, `${PluginType.DEPUBLISH}`].indexOf(
+        ongoing.currentPlugin!.pluginType
+      ) == -1 && !ongoing.cancelling
+    );
+  }
+
```
Comment (jochen-vermeulen) [inline src/app/dashboard/ongoingexecutions/ongoingexecutions.component.ts:60]: This function is not used (except in test code). I think it needs to be called from the html somewhere?

File: src/app/dataset/actionbar/actionbar.component.ts [hunk 2]
```diff
@@ -142,6 +144,16 @@ export class ActionbarComponent {
...
   }
 
+  /** canCancelWorkflow
+  /*  calculate if the workflow can be cancelled
+  */
+  canCancelWorkflow(): boolean {
+    return (
+      [`${PluginType.PUBLISH}`, `${PluginType.DEPUBLISH}`].indexOf(`${this.currentPluginName}`) ==
+        -1 && !this.isCompleted
+    );
```
Comment (jochen-vermeulen) [inline src/app/dataset/actionbar/actionbar.component.ts:154]: Seems like duplicate code - you added this check twice. Is there a possibility to have one source of truth somewhere?

File: src/app/dashboard/ongoingexecutions/ongoingexecutions.component.ts (hunk 1 above)
Comment (andyjmaclean) [reply src/app/dashboard/ongoingexecutions/ongoingexecutions.component.ts]: Fixed

File: src/app/dataset/actionbar/actionbar.component.ts (hunk 2 above)
Comment (andyjmaclean) [reply src/app/dataset/actionbar/actionbar.component.ts]: It was similar - so making the helper wasn't painless.

**Pull request merged**